test/e2e/07_add_custom_token.spec.js

```diff
@@ -38,6 +38,13 @@ const customTokensDetails = [
     symbol: 'SUSHI',
     decimal: '18'
   },
+  {
+    chain: 'avalanche',
+    address: '0x0b9d5d9136855f6fec3c0993fee6e9ce8a297846',
+    name: 'LINK',
```

Comment:
name and symbol :D

Reply:
ohhh sorry - will change it :)

Reply:
I fixed that - basically this is for the test cases and this is the token Link on avalanche, but now I added another token Link.e - as this is its symbol on avalanche and I also updated the address.